### PR TITLE
Addded Homepage

### PR DESCRIPTION
**Pull request by ayush65**
-> Converted html to jsx 
-> Divided the page into functional components

### CODE REVIEW

File: src/Homepage/category-card/Categorycard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  useEffect(() => getCategories(), []);
+
+  return (
+    <div className="Categorycard">
+  <h1 className="Categorycard-heading">Categories</h1>
+  <div className="card-grid">{
+            data.map((item) => {
+              return <div className="card card-shadow">
+                <div className="card-header card-image">
+                  <img src={item.img}/>
```
Comment (NtshVrm) [inline src/Homepage/category-card/Categorycard.js:30]: do add alt attributes in your img tags

**Review by NtshVrm — APPROVED** [review]
Add your package files to gitignore and add alt in img tags.
Other than the suggested changes:

- File Structure is clean
- Nicely broken down into components

File: src/App.css [hunk 2]
```diff
@@ -1,51 +0,0 @@
...
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #22201c;
```
Comment (HarshithaKonduru214) [inline src/App.css:17]: Pls add all the colors into variables.

File: src/App.css [hunk 3]
```diff
@@ -1,51 +0,0 @@
...
-.brand-description {
-  font-size: 1.4rem;
-}
-
-.links {
-  display: flex;
-}
-
-.links a {
-  color: #d9d9d9;
```
Comment (HarshithaKonduru214) [inline src/App.css:48]: The above comment applies to this entire file.

File: src/App.js [hunk 4]
```diff
@@ -1,41 +1,10 @@
 import "./App.css";
-import logo from "./logo.png";
+import Homepage from "./Homepage/Homepage";
```
Comment (HarshithaKonduru214) [inline src/App.js:2]: Can add barrel imports as the components and pages increase.

File: src/Homepage/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+function Footer() {
+  return (
+    <div className="Footer-container">
+    
+        <div className="Footer">
+        <div class="footer-col">
+  	 			<ul className="footer-links footer-align">
+  	 				<li><a  className="footer-links" href="#">About us</a></li>
+  	 				<li><a className="footer-links" href="#">Services</a></li>
+  	 				<li><a className="footer-links" href="#">privacy policy</a></li>
```
Comment (HarshithaKonduru214) [inline src/Homepage/Footer/Footer.js:13]: Can add link tags instead of anchor

File: src/Navbar/Navbar.js [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+import { GiHamburgerMenu } from "react-icons/gi";
+
+function Navbar() {
+    const [showMediaIcons, setShowMediaIcons] = useState(false);
+
+  return (
+    <>
+     <div className="navbar-container">
+    
+
```
Comment (HarshithaKonduru214) [inline src/Navbar/Navbar.js:13]: Take care of unnecessary spaces.

**Review by HarshithaKonduru214 — APPROVED** [review]
Added a few comments. Liked the fact that you broke down to neat components.

File: src/Homepage/category-card/Categorycard.js (hunk 1 above)
Comment (ayush65) [reply src/Homepage/category-card/Categorycard.js]: hanks for the review . i have done, the changes which you have suggested

File: src/App.css (hunk 2 above)
Comment (ayush65) [reply src/App.css:17]: that was the deleted code from the App.css

File: src/App.js (hunk 4 above)
Comment (ayush65) [reply src/App.js]: thansk for the suggestion i have implemented that .

File: src/Navbar/Navbar.js (hunk 6 above)
Comment (ayush65) [reply src/Navbar/Navbar.js]: yea i have done the changes as you have suggested , Thanks for pr review .

**Pull request merged**